mcsema/BC/Instruction.cpp

```diff
@@ -84,6 +84,21 @@ static llvm::Value *LoadAddressRegVal(llvm::BasicBlock *block,
   return value;
 }
 
+static bool IsFramePointerReg(
+            const remill::Operand::Register &reg) {
+
+  if (reg.name.empty()) {
+    return false;
+  }
+
+  if (mcsema::gArch->IsAMD64()) {
+    return !std::strcmp(reg.name.c_str(), "RBP");
+  } else if (mcsema::gArch->IsX86()) {
```

Comment:
If it is possible that the string might be "ebp" or "rbp" (lowercase) then use `stricmp` here.

If `reg.name` is always uppercase, though, why not use the comparison operator? `return reg.name != "EBP"`

Reply:
`RBP/EBP` will always be uppercase. Replaced `strcmp` with the comparison operator.